airbyte-integrations/connectors/source-tmdb/source_tmdb/tmdb.yaml

```diff
@@ -0,0 +1,295 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: []
+  requester:
+    url_base: "https://api.themoviedb.org/3/"
+    http_method: "GET"
+    authenticator:
+      type: ApiKeyAuthenticator
+      header: "apikey"
+      api_token: "{{ config['api_key'] }}"
+  retriever:
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    paginator:
+      type: NoPagination
+    requester:
+      $ref: "*ref(definitions.requester)"
+  base_stream:
+    schema_loader:
+      type: JsonSchema
+      file_path: "./source_tmdb/schemas/{{ options['name'] }}.json"
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+
+  certification_movie_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "certification_movie"
+      path: "/certification/movie/list?api_key={{ config['api_key'] }}"
```

Comment:
@btkcodedev why are you adding the api_key here and in the authentication? Keep only the Auth method.

Reply:
Done